samples/test/steps/given.ts

```diff
@@ -40,6 +45,11 @@ Given(
   navigateToLoginAndAuthenticate
 );
 
+Given(
+  /^a User named "([^/s]+)" created in the admin interface with a Password only$/,
+  _ => ({}) // todo: createAndStoreUserInContext
```

Comment:
todo: use code in [this PR](https://github.com/okta/okta-auth-js/pull/782/files#diff-0ba77150dd83ad840eafe3f8afad07d6b889f3dede7b679df363a391e7470387R3)